utils/api.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "https://particle.onrender.com/api",
+});
+
+export const fetchArticles = () => {
+    return api.get("/articles").then((res) => {
+      return res.data;
```

Comment:
If you return res.data.articles here then you won't need to deal with it in the useEffect. 

Nice work with axios, I would rename api to something more clear though!